pkgs/top-level/aliases.nix

```diff
@@ -42,6 +42,7 @@ mapAliases ({
   aircrackng = aircrack-ng; # added 2016-01-14
   ammonite-repl = ammonite; # added 2017-05-02
   arduino_core = arduino-core;  # added 2015-02-04
+  arduino_mk = arduino-mk;  # added 2018-12-14
```

Comment:
probably shouldn't have this -- this aliases file is just for backwards compatibility.